pkg/wfc3/lib/wf3rej/readpar.c

```diff
@@ -240,7 +240,9 @@ int rejpar_in (clpar *par, int newpar[], int nimgs, float exptot, int *niter,
         trlmessage (MsgText);
         sprintf (MsgText," propagation threshold = %0.1f", par->thresh);
         trlmessage (MsgText);
-        sprintf (MsgText," scale noise = %0.1f%%", par->scalense);
+        // Double-escape for trlmessage. Direct trlmessage calls don't require double-escaping.
+        // This will be a direct trlmessage call in the future.
+        sprintf (MsgText," scale noise = %0.1f%%%%", par->scalense);
```

Comment:
I am not sure why this routine did not get updated.

Reply:
Oh, because this lives in wfc3 and everything is calling the common trl functions. I left the other instruments alone unless I had to touch them to make them compatible.

Reply:
Yes. About that... Initially I converted the `sprintf`s everywhere, however due to the size of the patch, @pllim and I decided I should reduce the scope to ACS to start with. In the original changeset all references to the `asn` and `cte` messaging functions were removed and replaced with direct calls to `trl(error|message|warn)`.

I'll be putting in PRs for the other instruments and library code in the coming days.

Reply:
Indeed. And I still plan to test this manually when I can get hstcal to compile on my VM. 😭